lib/screens/feed_screen.dart

```diff
@@ -28,11 +28,14 @@ class _FeedScreenState extends State<FeedScreen> {
         title: Text("Feed Screen"),
       ),
       body: RefreshIndicator(
+        backgroundColor: Theme.of(context).backgroundColor,
         onRefresh: _reloadProducts,
         child: PaginateFirestore(
-          itemBuilderType: PaginateBuilderType.listView,
+          itemBuilderType: PaginateBuilderType.gridView,
           itemBuilder: (index, ctx, documentSnapshot) => ProductItem(
-            documentSnapshot.data['title'],
+            title: documentSnapshot.data['title'],
+            size: "plusSize",
+            brand: "Eldordado",
```

Comment:
Replace these temp strings with placeholder names e.g. <size> or PLACEHOLDER: BRAND together with a TODO. This is not very important but it can cause confusion for new developers in the project.